src/test/scala/chiselTests/FixedIOModuleSpec.scala

```diff
@@ -20,10 +20,10 @@ class FixedIOModuleSpec extends ChiselFlatSpec with Utils with MatchesAndOmits {
     class Foo extends FixedIORawModule[Bool](Bool()) {
       val a = IO(Bool())
     }
-    val exception = intercept[IllegalArgumentException] {
-      ChiselStage.emitCHIRRTL(new Foo)
+    val exception = intercept[ChiselException] {
+      ChiselStage.emitCHIRRTL(new Foo, Array("--throw-on-first-error"))
```

Comment:
Nit: unnecessary:
```suggestion
      ChiselStage.emitCHIRRTL(new Foo)
```

Reply:
Unfortunately, it isn't unnecessary because I think chisel catches the error, then throws one big error aggregating the messages :(